### ISSUE TITLE
Design mobile app UI/UX

### ISSUE THREAD

**Issue by Sevenarth** [healthyhomeuk/healthyhome#7]
Before the development on the mobile app starts, a design of the UI/UX of the mobile app must be made. This will guide the app developers throughout the development.

### UI/UX requirements
- User-friendly, minimalistic and well-organised. Avoid clutter at all costs;
- A real-time/historical snapshot facade of the current overall IAQ index, with a selection of major measurements in the background. The index must be accompanied with a brief description, with a link to a view providing detailed information on the levels. It must also be accompanied with some essential tips for the improvement of the conditions if needed. This should be adaptive, meaning that it must consider all the sensorial variables;
- A list of devices with a device registration function;
- A device page showing information on firmware version, online/offline status, last seen timestamp, reset and clear data functions;
- A timeline to analyse historical data;
- A prompt at first startup that guides the user into registering the first device.

The design could be made using software such as Adobe XD.

**Comment by amrllkmn** [comment]
I agree with UI/UX requirements. I've found some articles that could help guide us with the design:

[10 essential UI (user-interface) design tips](https://webflow.com/blog/10-essential-ui-design-tips)
[A Comprehensive Guide To Mobile App Design](https://www.smashingmagazine.com/2018/02/comprehensive-guide-to-mobile-app-design/)

Once we have a complete set of user stories, we can set up some wireframes. This would make it easier for us to play around with how the user will navigate the app.

**Comment by amrllkmn** [comment]
Hey @Sevenarth, @roryort, I'm attaching an image of the wireframes here. What do you guys think? I know that for the Particle Concentration the unit's not right but I'm just making sure that this is how we will guide the users to improve the environment.

![Prototype](https://user-images.githubusercontent.com/35376649/108910689-1f3b2b80-761e-11eb-8c7c-ac7bf4a7e116.png)

**Comment by roryort** [comment]
> Hey @Sevenarth, @roryort, I'm attaching an image of the wireframes here. What do you guys think? I know that for the Particle Concentration the unit's not right but I'm just making sure that this is how we will guide the users to improve the environment.
> 
> ![Prototype](https://user-images.githubusercontent.com/35376649/108910689-1f3b2b80-761e-11eb-8c7c-ac7bf4a7e116.png)

Hi Amir, these are looking good. I have a few comments.

Firstly, I am not sure about the slight brown colour that you are using in some of the cards. Is there a different colour that could be used? Sorry, I can't remember if we discussed this in our last meeting.

Secondly, there is some wording that I think could be changed a little. For example, instead of "Unacceptable Conditions" maybe something along the lines of "Poor Conditions" could be used?

Other than my comments above it all looks really good!

**Comment by Sevenarth** [comment]
Looking good! I agree with Rory and also I wanted to add that the unit of measurements should be on a separate line than the measurement. The original problem was to account for overflows, and I believe that leaving it on the same line did not solve the problem?

**Comment by amrllkmn** [comment]
Okay, so I fixed the colours, And I moved the line of the unit of measurements to the side of the labels that way the measurements obtained has more space. In addition to that, I've also changed the naming of the conditions of indoor air quality. Plus, I've made the historical reading of the data. Plus just a simple splash screen for when the app is first opened.

![image](https://user-images.githubusercontent.com/35376649/109201267-8b8e6a00-7799-11eb-9a0a-e718543e79bf.png)
